From a18b9d4b0b0fca2ef4aaff19285bb87333c01ff7 Mon Sep 17 00:00:00 2001
From: Manohar Castelino <manohar.r.castelino@intel.com>
Date: Thu, 27 Dec 2018 09:19:48 -0800
Subject: [PATCH] firecracker: Network API update

The Network API no longer supports the state field. Using the
state field causes the API call to fail.

Note: The firecracker binary needs to be updated to match the API
as the older versions will expect the state field.

Signed-off-by: Manohar Castelino <manohar.r.castelino@intel.com>
---
 virtcontainers/fc.go | 1 -
 1 file changed, 1 deletion(-)

diff --git a/virtcontainers/fc.go b/virtcontainers/fc.go
index 6ba67c001..573ad88d4 100644
--- a/virtcontainers/fc.go
+++ b/virtcontainers/fc.go
@@ -525,7 +525,6 @@ func (fc *firecracker) fcAddNetDevice(endpoint Endpoint) error {
 		GuestMac:          endpoint.HardwareAddr(),
 		IfaceID:           &ifaceID,
 		HostDevName:       endpoint.NetworkPair().TapInterface.TAPIface.Name,
-		State:             "Attached",
 	}
 	cfg.SetBody(ifaceCfg)
 	cfg.SetIfaceID(ifaceID)
